### DISCUSSION TITLE
bug(MatCalendar): @Input("startAt ") is not referenced for changes

### DISCUSSION THREAD

**Discussion by hhsissi** [angular/components#25340] [Ideas]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

@Input("startAt") is not referenced for changeDetection.

[https://github.com/angular/components/blob/a378147356e2b6a43fa642ecbc69e37f3077fac7/src/material/datepicker/calendar.ts#L411](https://github.com/angular/components/blob/a378147356e2b6a43fa642ecbc69e37f3077fac7/src/material/datepicker/calendar.ts#L411)

There is a workaround with "activeDate", but the problem is it makes a bit heavy on developers to implement programmatic month change (as it it not useable through templates). using startAt would make ours lives much easier when it comes to changing month view.

### Reproduction

Steps to reproduce:
1. Create a component with a MatCalendar
2. set mat-calendar "startAt" attribute in template and bind it to a property in your component (startAt) for example.
3. change programmaticaly the startAt to another month
4. Nothing happens (and this is a problem)

### Expected Behavior

startAt updates should be referenced for changeDetection and trigger a view update.

### Actual Behavior

Changing startAt value won't trigger change detection.

A workaround is changing the "activeDate", that is not referenced as an Input, therefore developers will have a hard time finding it.
Most people are actualy using internal functions like "calendar.monthView._init()" and stuff like that...

### Environment

- Angular: 14.0.4
- Material: 14.0.4 and previous

**Comment by zarend** [comment]
Hello @hhsissi ,

Since "startAt" is for the date that the calendar opens to initially, there is no requirement that changing startAt after the calendar opens should have an effect. I'll convert this to a discussion so more developers could chime in on how to implement programmatic month change. perhaps there could be a better way to do that.

-Zach

**Reply by hhsissi** [reply]
Hi @zarend , 

Thanks for you reply / time.

Well my suggestion is already set, let's wait & see some other suggestions.
